ecpy_hqc_legacy/tasks/tasks/instr/synthHD_tasks.py

```diff
@@ -0,0 +1,458 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015-2016 by EcpyHqcLegacy Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""Task perform measurements with a synthHD.
```

Comment:
This is inaccurate. Should be:
"""Interface for the synthHD microwave source.

"""